packages/devtools/initializers/create-bare/project.json

```diff
@@ -2,6 +2,14 @@
   "sourceRoot": "packages/devtools/initializers/create-bare/src",
   "projectType": "library",
   "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
```

Comment:
I think this isn't needed because it'll now inherit from nx.json where this is specified

Reply:
actually a lot of other packages just have `"build": {},` that might even be enough with inheritance if it follows the patterns